package.json

```diff
@@ -34,6 +34,10 @@
     "babel-eslint": "^10.1.0",
     "classnames": "^2.2.5",
     "core-js": "^3.6.5",
+    "dotenv": "^8.2.0",
+    "dotenv-webpack": "^5.1.0",
+    "eslint": "^7.14.0",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
```

Comment:
Should these be devDependencies since they are not used in production build?

Reply:
Yes... I thought I had moved them, will do it now.